src/viam/components/board/board.py

```diff
@@ -38,11 +38,23 @@ class Analog:
         name: str
         """The name of the analog pin"""
 
+        class Value:
```

Comment:
You might want to make this a `@dataclass` to automatically generate an initializer